types/index.d.ts

```diff
@@ -12,9 +12,7 @@ export {
 } from "./helpers";
 
 export {
-  DefineGetters,
```

Comment:
IMHO it doesn't bother if we re-export other utility type helpers. These helper types can be convenient for users to separate getters/actions/mutations to different files.

Reply:
Makes sense. I've exposed them again.